spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraRelation.scala

```diff
@@ -42,7 +42,7 @@ private[cassandra] case class CassandraRelation
 
 object ColumnDataType {
 
-  private val primitiveTypeMap = Map[connector.types.ColumnType[_], types.DataType](
```

Comment:
Although it wasn't changed in this PR, there is probably a bug - the Connector type `connector.types.BlobType` is mapped to the Catalyst type `types.Bytes` instead of `types.Binary`.